source/2019-09-27-the-ember-times-issue-117.md

```diff
@@ -26,80 +32,96 @@ responsive: true
 
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Accessibility, Compiler Secrets and Modern Ember @ EmberCamp 🐹](#section-url)
+
+
+Last week the Ember community went on another extraordinary trip into the metropolitan wilderness of Illinois:
```

Comment:
I think you used "another" to indicate that EmberCamp is in its 2nd year. However, since there isn't a mention of the year, I thought "an" may be better.
```suggestion
Last week, the Ember community went on an extraordinary trip into the metropolitan wilderness of Illinois:
```